src/common/underline.js

```diff
@@ -0,0 +1,13 @@
+const underline = (e) => {
+  if (e.currentTarget.checked === true) {
+    const index = e.currentTarget.parentNode.childNodes[0].innerHTML;
+    console.log(index);
```

Comment:
- [ ] Kindly make sure you remove all the `consloe.log()` in your code vase for **[JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)**. `console.log()` is ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.

Reply:
removed the same